src/audio/buffer.c

```diff
@@ -303,30 +292,7 @@ void comp_update_buffer_consume(struct comp_buffer __sparse_cache *buffer, uint3
 void buffer_attach(struct comp_buffer *buffer, struct list_item *head, int dir)
 {
 	struct list_item *list = buffer_comp_list(buffer, dir);
-	struct list_item __sparse_cache *needs_sync;
```

Comment:
FIXME: this is still a
 * problem with different cores.

after change, is there still a problem?

Reply:
I don't know what was the problem and if it disappeared or not, so I left the comment just in case

Reply:
after change, there is almost no code left in this function, ok, let's keep it, :)